core/web/router.go

```diff
@@ -384,11 +384,16 @@ func uiCorsHandler(config store.Config) gin.HandlerFunc {
 
 func readBody(reader io.Reader) string {
 	buf := new(bytes.Buffer)
-	buf.ReadFrom(reader)
+	_, err := buf.ReadFrom(reader)
+	if err != nil {
+		logger.Warn("unable to read from body for sanitazation: ", err)
```

Comment:
sanitization

Reply:
This has been fixed. If you refresh the changes tab, you should see the correct spelling.